python/cudf_polars/tests/test_scan.py

```diff
@@ -10,17 +10,7 @@
 
 
 @pytest.fixture(
-    params=[
-        (None, None),
-        pytest.param(
-            ("row-index", 0),
-            marks=pytest.mark.xfail(reason="Incorrect dtype for row index"),
-        ),
-        pytest.param(
-            ("index", 10),
-            marks=pytest.mark.xfail(reason="Incorrect dtype for row index"),
-        ),
-    ],
+    params=[(None, None), ("row-index", 0), ("index", 10)],
```

Comment:
If possible, can we add a unit-test that that explicitly tests for this?

(maybe a no-op case that doesn't round-trip correctly)

Reply:
Nevermind, I misunderstood what that test did.